shelley/chain-and-ledger/shelley-spec-ledger-test/test/Test/Shelley/Spec/Ledger/NonTraceProperties/Generator.hs

```diff
@@ -95,13 +95,20 @@ import Test.Shelley.Spec.Ledger.Generator.Core
   ( applyTxBody,
     genesisCoins,
   )
-import Test.Shelley.Spec.Ledger.Generator.Utxo (splitCoin)
 import Test.Shelley.Spec.Ledger.NonTraceProperties.Mutator
 import Test.Shelley.Spec.Ledger.NonTraceProperties.Validity
 import Test.Shelley.Spec.Ledger.Orphans ()
 import Test.Shelley.Spec.Ledger.Utils
 import Unsafe.Coerce
 
+-- ==============================================================
```

Comment:
(will have to move this elsewhere and remove NonTraceProperties)

Reply:
In another PR